qiskit/visualization/gate_map.py

```diff
@@ -705,24 +908,78 @@ def plot_error_map(backend, figsize=(12, 9), show_title=True):
 
     color_map = sns.cubehelix_palette(reverse=True, as_cmap=True)
 
-    props = backend.properties().to_dict()
-    config = backend.configuration().to_dict()
-
-    num_qubits = config["n_qubits"]
-
-    # sx error rates
-    single_gate_errors = [0] * num_qubits
-    for gate in props["gates"]:
-        if gate["gate"] == "sx":
-            _qubit = gate["qubits"][0]
-            for param in gate["parameters"]:
-                if param["name"] == "gate_error":
-                    single_gate_errors[_qubit] = param["value"]
-                    break
-            else:
-                raise VisualizationError(
-                    f"Backend '{backend}' did not supply an error for the 'sx' gate."
-                )
+    backend_version = _get_backend_interface_version(backend)
+    if backend_version <= 1:
+        num_qubits = backend.configuration().n_qubits
+        cmap = backend.configuration().coupling_map
+        props = backend.properties().to_dict()
+        single_gate_errors = [0] * num_qubits
+        read_err = [0] * num_qubits
+        cx_errors = []
+        # sx error rates
+        for gate in props["gates"]:
+            if gate["gate"] == "sx":
+                _qubit = gate["qubits"][0]
+                for param in gate["parameters"]:
+                    if param["name"] == "gate_error":
+                        single_gate_errors[_qubit] = param["value"]
+                        break
+                else:
+                    raise VisualizationError(
+                        f"Backend '{backend}' did not supply an error for the 'sx' gate."
+                    )
+        if cmap:
+            directed = False
+            if num_qubits < 20:
+                for edge in cmap:
+                    if not [edge[1], edge[0]] in cmap:
+                        directed = True
+                        break
+
+            for line in cmap:
+                for item in props["gates"]:
+                    if item["qubits"] == line:
+                        cx_errors.append(item["parameters"][0]["value"])
+                        break
+                else:
+                    continue
```

Comment:
I see this was here originally, but is there any point to this `continue`? As I understand it, it should advance the outer loop (`for line in map`), but there's nothing left to do anyway.

Reply:
yeah there is no reason to have a continue here, I just copied this from what was here before, but I'll just drop this in the next revision

Reply:
Dropped in: https://github.com/Qiskit/qiskit-terra/pull/7814/commits/fac6ffb169a60e147e0340c68c2f570fdcffcc1b